src/services/identity/UsersService.ts

```diff
@@ -2,24 +2,40 @@ import { Op, ModelStatic } from "sequelize"
 import { Sequelize } from "sequelize-typescript"
 import { RequireAtLeastOne } from "type-fest"
 
-import { Repo, Site, User, Whitelist, SiteMember } from "@database/models"
+import { Otp, Repo, Site, User, Whitelist, SiteMember } from "@database/models"
+import { BadRequestError } from "@root/errors/BadRequestError"
+import { milliSecondsToMinutes } from "@root/utils/time-utils"
 import SmsClient from "@services/identity/SmsClient"
 import TotpGenerator from "@services/identity/TotpGenerator"
 import MailClient from "@services/utilServices/MailClient"
 
+import OtpService from "./OtpService"
+
+const { OTP_EXPIRY } = process.env
+
+const PARSED_EXPIRY = parseInt(OTP_EXPIRY!, 10) ?? undefined
+
+const MAX_NUM_OTP_ATTEMPTS =
+  parseInt(process.env.MAX_NUM_OTP_ATTEMPTS!, 10) ?? 5
```

Comment:
nit: could we inline the `process.env.MAX_NUM_OTP_ATTEMPTS` to the `process.env` destructuring above?

Reply:
okay sure!